.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+#### Expected behavior vs actual behavior
+<br><br>
+
+#### Steps to reproduce
+*(e.g., detailed walkthrough, runnable script, example application)*
+<br><br><br>
+
+#### Environment
+
+AMS Version *(commit ref if not on tag)*: 
```

Comment:
No objections to `ActiveModelSerializers`. I figured it could be picked up under Additional helpful information if they provide Gemfile.lock. If this ever became too complicated and we definitely need base information to cut down on support burden, we could make a script for people to download and run, then copy and paste the output into that area of the issue. I figured we could revisit it if it got to that point.